src/test/java/org/numenta/nupic/serialize/HTMObjectInputOutputTest.java

```diff
@@ -22,6 +23,7 @@
 import org.numenta.nupic.network.sensor.SensorParams;
 import org.numenta.nupic.network.sensor.SensorParams.Keys;
 import org.numenta.nupic.util.FastRandom;
+import static org.numenta.nupic.network.NetworkTestHarness.*;
```

Comment:
@Hopding  I think the convention is to list static imports at the top, above regular imports? Can you group this one with the others at the top?